src/test_linked_list.py

```diff
@@ -88,6 +98,14 @@ def test_remove(initial, data, result):
     assert test_case.remove(test_case_node).data == result
 
 
+def test_remove_empty():
+    """Test remove method against an empty list."""
+    test_case = LinkedList()
+    with pytest.raises(IndexError) as message:
+        test_case.remove("taco")
+    assert "That value is not in the list." in str(message)
```

Comment:
I'll reiterate, what about removing something from a populated list that isn't there?